CHANGELOG.unreleased.md

```diff
@@ -17,6 +17,7 @@
 
 - Fix mounting of NFS remote in FreeBSD (PR [#4988](https://github.com/vatesfr/xen-orchestra/issues/4988))
 - [Remotes] Fix "remote is disabled" error on getting the remotes info (commit [eb2f429964d7adc264bf678c37e49a856454388e](https://github.com/vatesfr/xen-orchestra/commit/eb2f429964d7adc264bf678c37e49a856454388e))
+- [SDN Controller] Fix encrypted tunnels recreation [#4996](https://github.com/vatesfr/xen-orchestra/pull/4996)
```

Comment:
What was the issue?

Reply:
When an host rebooted for instance, only half of the tunnel was recreated which is problematic in the case of encrypted tunnels because the tunnels password are no more synced.

Reply:
IMHO, a description of a fix which say `fix foobar` is not really useful.

It should described the problem that is fixed.

Reply:
What about, `No more broken encrypted tunnels after host reboot`

Reply:
It's better :slightly_smiling_face: 

```suggestion
- [SDN Controller] Broken encrypted tunnels after host reboot [#4996](https://github.com/vatesfr/xen-orchestra/pull/4996)
```

Reply:
Did you intend to remove the `No more`?

Reply:
As you wish, it works with or without it.

It's in the bug fixes section, I believe it's implied that we fixed it and not implemented it :-p